src/pages/pass-finalisation/cat-adi-part2/pass-finalisation.cat-adi-part2.page.ts

```diff
@@ -128,7 +128,7 @@ export class PassFinalisationCatADIPart2Page extends BasePageComponent {
     Object.keys(this.form.controls).forEach(controlName => this.form.controls[controlName].markAsDirty());
     if (this.form.valid) {
       this.store$.dispatch(new PersistTests());
-      this.navController.push(CAT_ADI_PART2.HEALTH_DECLARATION_PAGE);
+      this.navController.push(CAT_ADI_PART2.BACK_TO_OFFICE_PAGE);
```

Comment:
Should we be removing the views here like is done in the HEALTH_DECLARATION_PAGE?